extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/SmallRyeGraphQLRuntimeConfig.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.smallrye.graphql.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "smallrye-graphql", phase = ConfigPhase.RUN_TIME)
+public class SmallRyeGraphQLRuntimeConfig {
+
+    /**
+     * Deactivate the graphql schema and graphql-ui endpoint.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean deactivate;
```

Comment:
I was thinking about this yesterday and my first inclination was to have `expose` defaulting to true.

Just a proposal, I don't know what's best really :).